pytaskflow/dashboard/controllers/jobs.py

```
@@ -0,0 +1,101 @@
+from pytaskflow.client import Client
+
+PAGE_SIZE = 20
+
+
+def get_client(state: State) -> Client:
+    return state.client
+
+
+class JobsController(Controller):
+    path = "/jobs"
+    dependencies = {"client": Provide(get_client, sync_to_thread=False)}
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)